eng/Directory.Packages.props

```diff
@@ -108,7 +108,7 @@
     <PackageVersion Include="Microsoft.VisualStudio.Threading" Version="17.9.3-alpha" />
     <PackageVersion Include="Microsoft.VisualStudio.Utilities.Internal" Version="16.3.56" />
     <PackageVersion Include="Nerdbank.Streams" Version="2.10.69" />
-    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageVersion Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
```

Comment:
This might need to be conditional on source build. Otherwise we might get conflicts with VSSDK

Reply:
How would it conflict? Not saying you're wrong, I just don't have a sense of the systems here.

Reply:
Let's say source build has lower version than VSSDK. Then we end up downgrading VS components that lower version.

Reply:
I imagine though we're not even building the VS components though in source build, right?

Reply:
Right. When building from source, the `NewtonsoftJsonVersion` property gets overridden by the source-build generated versions file. When not building from source, the `NewtonsoftJsonVersion` property gets used as-is.

Reply:
Correct